install/prerequisites.adoc

```diff
@@ -408,6 +405,44 @@ domain is not listed in the search list. Otherwise, containers managed by
 {product-title} might fail to resolve host names properly.
 ====
 
+[[node-hostnames-prereq]]
+==== Configuring Node Host Names
+
+When you set up a cluster that is not integrated with a cloud provider, you must
+correctly set your nodes' host names. Each node's host name must be resolvable,
+and each node must be able to reach each other node. 
+
+To confirm that a node can reach another node:
+
+. One one node, obtain the host name:
++
+----
+$ hostname
+
+master-1
```

Comment:
Since when? We've had **a lot** of issues regarding certificates not having the fqdn as a subject alternative name and connecting to them using the fqdn, or having only the fqdn in the alt names and services connecting to them using only the shortname.

This has been a mess, particuarly with the etcd_hostname variable. Not only it has been problematic, but also the vast majority of our customers have the ```hostname``` as the fqdn

Reply:
I understand it can return both the short name or the fqdn. I mean this implies hostname **must** return the shortname, and not only this is not necessary but also it's problematic.

We've had a lot of support cases related to having shortname  in a place and fqdn in anohter. I would say we should document hostname returning the fqdn as a best practice.

Reply:
Okay, I am fine with document hostname returning the fqdn as a best practice.